app/lib/collections/class.js

```diff
@@ -64,17 +66,18 @@ if (Meteor.isServer) {
 
     Class.allow({
         insert: function (userId, doc) {
-            return false;
+            return true;
```

Comment:
Huwag lang pong true. Paki check muna if the user is an admin. As in dito rin sa function na ito. Because not securing this means that I can insert if I make a different meteor method that skips the checks.